src/display/annotation_layer.js

```diff
@@ -2465,6 +2466,19 @@ class FileAttachmentAnnotationElement extends AnnotationElement {
  */
 
 class AnnotationLayer {
+  static _addToAccessibilityManager(element, id, div, accessibilityManager) {
```

Comment:
Given that the most important thing here is adding the annotation-element to the DOM, how about the following name instead to make that clearer (and if possible, let's make this method private)?
```suggestion
  static #appendElement(element, id, div, accessibilityManager) {
```